lib/std/os/windows.zig

```diff
@@ -1566,6 +1576,45 @@ pub fn GetEnvironmentVariableW(lpName: LPWSTR, lpBuffer: [*]u16, nSize: DWORD) G
     return rc;
 }
 
+// zig fmt: off
```

Comment:
I kept these inside for https://github.com/ziglang/zig/issues/14251, but they are generally useful for other users as well short of advanced list attributes.